wappsto/connection/seluxit_rpc.py

```diff
@@ -194,100 +193,40 @@ def get_rpc_state(
 
         """
         update = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%fZ')
+        if set_type == 'report':
```

Comment:
Why not just have the set_type be the correct type, so that you do not need to convert the string.